src/shared/components/download/Download.tsx

```diff
@@ -102,6 +116,47 @@ const Download = (props: DownloadProps<JobTypes>) => {
     { props, job, selectedColumns: columnNames },
     getDownloadInitialState
   );
+  const { data } = useDataApi<ReceivedFieldData>(
+    apiUrls.resultsFields(namespace)
+  );
+
+  const fieldData = useMemo(
+    () => prepareFieldData(data, undefined, true),
+    [data]
+  );
+
+  const getFieldsWithMultipleValueXref = useCallback(
```

Comment:
It seems this function returns a boolean rather than the actual fields so what do you think about using an `is` for the function prefix eg `isMultiValueCrossReferenceField`. Thinking about it `isMultiValueCrossReference` is a bit confusing to me (I know it's coming from the config file) but basically it's saying the xref has a full option available? If that's true what do you think then about `isXrefWithFullOption`? Just an idea! I'm trying to imagine reading this in a years time and trying to remind myself what `isMultiValueCrossReference` means.